src/main/java/org/apache/flink/benchmark/RemoteChannelThroughputBenchmark.java

```diff
@@ -48,6 +49,9 @@
 
     private MiniCluster miniCluster;
 
+    @Param({"AlignedCheckpoint", "UnalignedCheckpoint"})
```

Comment:
`ALIGNED`/`UNALIGNED` to shorten the benchmark name plus to be consistent with the existing parameters (`ORDERED`/`UNORDERED`, `ROCKS`/`MEMORY`/`FS_ASYNC`/...)?

Reply:
Actually I tried the shorten name in early version, but i am wondering it might confuse external users, so i changed to the semantic name finally. Anyway, i have no preference here and will change it as you suggested.

Reply:
There are no external users that I know of :)